stable/database/values.yaml

```diff
@@ -334,6 +334,7 @@ database:
       backupGroups: {}
       #  cluster0-0:
       #    labels: "pod-name sm-database-nuodb-cluster0-demo-hotcopy-0"
+      #    processFilter: "label(pod-name sm-database-nuodb-cluster0-demo-hotcopy-0)"
```

Comment:
How about providing a sample value that is not just the label filter, to give an example of when the user will want to use the option?

Reply:
I've intentionally added commented-out filter which matches the label selector to illustrate how the same use case is implemented either by `labels` or `processFilter`. Currently, we don't have a lot of distinct labels on SMs so that we can leverage fully the power of LBQuery syntax, however, there is a plan to "resurrect" the _placement_ labels like _zone_, _region_, _node_, etc.
I will take your comment into consideration when documenting backup groups in the user docs.